data_safe_haven/pulumi/dynamic/dsh_resource_provider.py

```diff
@@ -44,26 +44,21 @@ def partial_diff(
     def refresh(props: dict[str, Any]) -> dict[str, Any]:
         return dict(**props)
 
-    def check(self, old_props: dict[str, Any], new_props: dict[str, Any]) -> CheckResult:
```

Comment:
These signatures can't be changed as they're inherited from `pulumi.dynamic.ResourceProvider`.

Reply:
Fixed in 65280d45